docs/connector-development/config-based/understanding-the-yaml-file/record-selector.md

```diff
@@ -163,7 +253,7 @@ stream:
       - type: AddFields
         fields:
           - path: [ "start_date" ]
-            value: {{ stream_slice[ 'start_date' ] }}
+            value: { { stream_slice[ 'start_date' ] } }
```

Comment:
the extra space between the curly braces seems a little awkward instead of `{{`?